### ISSUE TITLE
pretrained model with video augmentation data?

### ISSUE THREAD

**Issue by JongMokKim** [NVIDIA/semantic-segmentation#6]
Hello! Thank you for nice work!!

I have some question about pretrained model you provide (cityscapes_best.pth)

in your paper, the model weight that get 81.5 miou (val) is trained by video augmentation data, sdc

but in here, github, training IOU of DeepWV3Plus get 81.4 IOU but tagged with "no sdc-aug"

my question is, which one is true? and is the provided pretrained model trained /wo sdc-aug? or /w sdc-aug?

Thank you in advance!

**Comment by bryanyzhu** [comment]
In our paper, the 81.5 score is obtained on cv0 split and use ResNeXt50 model. In the table here, the score 81.4 is obtained on cv2 split and use WideResNet38 model. Both the validation set and the model are not the same, so the scores are not directly comparable. Sorry for the confusion, we will provide a more comprehensive table for clear comparison. 

In general, with sdc-aug will give you better performance and better generalization capability than without sdc-aug. 

Right now, the provided `cityscapes_best.pth` model is trained with sdc-aug using WideResNet38 backbone. And it is trained on cv2 split to obtain best performance on the test leaderboard. Hope my explanation is clear, please let me know if you have more questions.

**Comment by JongMokKim** [comment]
Thank you for your answer, and it was clear to me

any way, can you provide wideresnet38 weight only /w mapillary pretrained and not cityscape? 
I just want to train from that point (train cityscape data)

**Comment by bryanyzhu** [comment]
Hi @karansapra, Is there a possibility that we release those Mapillary pretrained weights? Thank you.

**Comment by karansapra** [comment]
Let me look into it. We can definitely look at release the Wideresnet38 weights to train mapillary.

**Comment by karansapra** [comment]
Hi @JongMokKim  and @bryanyzhu  I have made a new commit that adds imagenet weights for WR38. 

https://github.com/NVIDIA/semantic-segmentation/commit/85203a701ec8ee88d0fc1595a65c56b72fb3c36c

**Comment by yangwangx** [comment]
Hi, @bryanyzhu and @karansapra , thanks for your great work!

Within the "cityscapes_best.pth" model, I find a command line. 

``` bash
> print( torch.load('pretrained_models/cityscapes_best.pth')['command'] )
```
``` text
--dataset cityscapes --cv 2 --arch network.deepv3.DeepWV3Plus --sgd --lr 0.002 --scale_min 0.5 --scale_max 2.0 --gblur --color_aug 0.25 --class_uniform_pct 0.5 --class_uniform_tile 1024 --bs_mult 2 --lr_schedule scl-poly --poly_exp 1.0 --max_epoch 180 --crop_size 800 --syncbn --coarse_boost_classes 14,6,15,16,3,12,17,4 --max_cu_epoch 120 --maxSkip 3 --jointwtborder --wt_bound 1.0 --rlx_off_epoch 120 --repoly 1.5 --rescale 1.0 --strict_bdr_cls 5,6,7,11,12,17,18 --snapshot /home/dcg-adlr-yizhu-data.cosmos216/yizhu/pretrained/ImageSeg/best_mapillary_DeepWV3PlusS3_50.03.pth --ckpt /home/dcg-adlr-yizhu-output.cosmos229/run_logs_git_release --tb_path /home/dcg-adlr-yizhu-output.cosmos229/run_logs_git_release --exp cv2_skip3_wt1_relax_epoch180_run3
```

I assume this is the training script used to obtain `cityscapes_best.pth` model. 
The starting point for this script is **`best_mapillary_DeepWV3PlusS3_50.03.pth`**.
Is it possible to release this model as well? Or, is it unavailable due to privacy concerns?

**Comment by bryanyzhu** [comment]
@karansapra @yangwangx @JongMokKim  Due to license issue, we may not be able to release the Mapillary pre-trained models at this moment. You can follow our readme to train your own model. It should reproduce, although I realize that training on Mapillary takes quite some time. Sorry for the inconvenience.

**Comment by JongMokKim** [comment]
Thank you for your kind reply :)
